contracts/PurchaseOfferMarket.sol

```diff
@@ -170,20 +175,26 @@ contract PurchaseOfferMarket is Initializable, OwnableUpgradeable, IPurchaseOffe
         uint256 _deposit,
         uint16 _limit,
         uint256 _minimumAcceptHeight,
+        uint256 _minimumStakingAmount,
         uint256 _expireDate
     ) external {
         require(!(IEraManager(settings.getEraManager()).maintenance()), 'G019');
         require(_expireDate > block.timestamp, 'PO002');
         require(_deposit > 0, 'PO003');
         require(_limit > 0, 'PO004');
+
         IPlanManager planManager = IPlanManager(settings.getPlanManager());
         PlanTemplateV2 memory template = planManager.getPlanTemplate(_planTemplateId);
         require(template.active, 'PO005');
         require(template.priceToken == settings.getSQToken());
 
+        IIndexerRegistry indexerRegistry = IIndexerRegistry(settings.getIndexerRegistry());
+        require(_minimumStakingAmount >= indexerRegistry.minimumStakingAmount(), 'PO012');
```

Comment:
No check here should be fine. minimumStakingAmount = 0 means no restriction on indexer's staking amount.